alerting/src/main/kotlin/org/opensearch/alerting/settings/AlertingSettings.kt

```diff
@@ -156,5 +173,94 @@ class AlertingSettings {
             -1L,
             Setting.Property.NodeScope, Setting.Property.Dynamic
         )
+
+        val MAX_ACTIONS_ACROSS_TRIGGERS = Setting.intSetting(
+            "plugins.alerting.max_actions_across_triggers",
+            DEFAULT_MAX_ACTIONS_ACROSS_TRIGGERS,
+            -1, MaxActionsTriggersValidator(internalClient),
+            Setting.Property.NodeScope, Setting.Property.Dynamic
+        )
+        val TOTAL_MAX_ACTIONS_ACROSS_TRIGGERS = Setting.intSetting(
```

Comment:
is this the max number of actions per trigger? If so, rename is to `TOTAL_MAX_ACTIONS_PER_TRIGGERS` since the current name still means the same as `MAX_ACTIONS_ACROSS_TRIGGERS`